stdlib/database/mysql/src/test/resources/test-src/query/simple-params-query-test.bal

```diff
@@ -656,6 +710,80 @@ function queryTimestampStringInvalidParam() returns @tainted record {}|error? {
     return queryMysqlClient(sqlQuery);
 }
 
+function queryEnumStringParam() returns @tainted record {}|error? {
+    sql:ParameterizedString sqlQuery = {
+            parts: ["SELECT * from ENUMTable where enum_type=", ""],
+            insertions: ["doctor"]
+     };
+    return queryMysqlClient(sqlQuery);
+}
+
+type EnumResult record {|
+    int id;
+    string enum_type;
+|};
+
+function queryEnumStringParam2() returns @tainted record {}|error? {
+    sql:ParameterizedString sqlQuery = {
+            parts: ["SELECT * from ENUMTable where enum_type=", ""],
+            insertions: ["doctor"]
+     };
+    return queryMysqlClient(sqlQuery, resultType=EnumResult);
+}
+
+function querySetStringParam() returns @tainted record {}|error? {
+    sql:ParameterizedString sqlQuery = {
+            parts: ["SELECT * from SetTable where set_type=", ""],
+            insertions: ["a,d"]
+     };
+    return queryMysqlClient(sqlQuery);
+}
+
+function queryGeoParam() returns @tainted record {}|error? {
+    sql:ParameterizedString sqlQuery = {
+            parts: ["SELECT id, ST_AsText(geom) as geomText from GEOTable"],
+            insertions: []
```

Comment:
Is it possible to avoid setting an empty array for `insertions`, If insertions not required?
May be setting default value

Reply:
This is a temporary syntax based on this - https://docs.google.com/document/d/1F1Hlxc8rWmKmJKiWnqmeDOQ1XFckrmbeL5V8UxQAD3c/edit#heading=h.f4zz0m9kmsmf, corresponding spec issue - https://github.com/ballerina-platform/ballerina-spec/issues/442. Once this issues are addressed in the language will make necessary changes to support that syntax and hence this also will change.